addons/point_of_sale/static/src/app/payment/payment_interface.js

```diff
@@ -37,6 +38,16 @@ export class PaymentInterface {
         this.supports_reversals = true;
     }
 
+    /**
+     * Call this function to disable the automatic calling of
+     * send_payment_request upon selecting the payment method.
+     * This allows manual input of an amount before sending
+     * the request to the terminal.
+     */
+    disable_fast_payments() {
```

Comment:
You could do a getter, something like
```suggestion
    get fast_payments() {
        return true;
    }
```
that you would override in chirldren class if needed ? That would be cleaner that calling a method in the setup IMO